astropy/units/quantity.py

```diff
@@ -217,14 +217,14 @@ class Quantity(np.ndarray):
 
     Parameters
     ----------
-    value : number, `~numpy.ndarray`, `Quantity` object (sequence), str
+    value : number or `~numpy.ndarray` or `Quantity` object (sequence) or str
```

Comment:
Can we keep the first two `,` and just add the last "or"?

Reply:
We can, I've just been trying to stick to https://numpydoc.readthedocs.io/en/latest/format.html, which has all options separated by "or". We haven't been careful about it throughout the docs because it hasn't been important. It's not super important now, but perhaps a followup PR could further standardize our docstrings. My personal vote is to just follow NumPy.

Reply:
I'm confused - I don't see that stated in the link you give.  There seem to be no examples with multiple `or`... Overall, my tendency would be to be as close to regular english as possible.

Reply:
Regex finding all ":" + " or " + " or " to replace with ":" + ", " + ", or "

Reply:
✅